lib/src/encrypted.dart

```diff
@@ -8,10 +8,10 @@ class Encrypted {
 
   /// Creates an Encrypted object from a hexdecimal string.
   Encrypted.fromBase16(String encoded)
-      : _bytes = Uint8List.fromList(
-          List.generate(encoded.length,
-                  (i) => i % 2 == 0 ? encoded.substring(i, i + 2) : null)
-              .where((b) => b != null)
+      : assert(encoded.length % 2 == 0),
```

Comment:
Same as here: https://github.com/leocavalcante/encrypt/pull/195/files#r579363311

Maybe worth pulling out into a separate method and testing it?

Reply:
Same as above, I guess.